### PR TITLE
add version and help commandline options

### PR DESCRIPTION
**Pull request by HanYoung-uwu**
As requested by #602

### CODE REVIEW

**Comment by kelson42** [comment]
@HanYoung-uwu Thank you very much

File: src/main.cpp [hunk 1]
```diff
@@ -7,7 +7,8 @@
 #if QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
   #include <QWebEngineUrlScheme>
 #endif
-
+#define _STR(...) # __VA_ARGS__
+#define STR(X) _STR(X)
```
Comment (mgautierfr) [inline src/main.cpp:11]: We don't need this, it is already in `src/kiwixapp.h`

**Review by mgautierfr — CHANGES_REQUESTED** [review]
Just a small change. Everything else is good.

**Comment by HanYoung-uwu** [comment]
force pushed

**Pull request merged**